azure-pipelines.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       export PATH=$HOME/.sonar/build-wrapper-linux-x86:$PATH
       autoreconf --install
       ./configure
-      build-wrapper-linux-x86-64 --out-dir build_wrapper_output_directory make clean all
+      build-wrapper-linux-x86-64 --out-dir bw_output make clean all
```

Comment:
I think we should port this change to other azure scripts.

Reply:
I'll remove this change to keep consistency with other examples